source/turtles/turtle_hero.py

```diff
@@ -0,0 +1,137 @@
+import pygame, random
+
+GREEN = (20, 255, 140)
+GREY = (210, 210, 210)
+WHITE = (255, 255, 255)
+RED = (255, 0, 0)
+PURPLE = (255, 0, 255)
+BLACK = (0, 0, 0)
+WIDTH = 100
+HEIGHT = 50
+ACC = 600
+TICK = 1/60.0
+SPEED_FAST = 300
+SPPED_SLOW = 150
+
+class Turtle(pygame.sprite.Sprite):
+    # This class represents a turtle. It derives from the "Sprite" class in Pygame.
+
+    def __init__(self, type, size_coeff, name, position):
+        """
+        :param type: turtle type (TODO: define)
+        :param size_coeff: SIZE_COEFF
+        :param name: name strinf
+        :param position: creation postition in pixels - tuple
```

Comment:
...tuple (x,y)? -> warto to wyspecyfikować, bo z kolejnością x,y zawsze jest problem w numpy, opencv itp